api/swagger.yaml

```diff
@@ -195,6 +195,29 @@ definitions:
       Propagation:
         type: "string"
 
+  ContainerNode:
+    type: "object"
+    description: "Node container is running on in Swarm"
```

Comment:
Could you use the description that's currently used on the type defined in Go? https://github.com/moby/moby/blob/39e6def2194045cb206160b66bf309f486bd7e64/api/types/types.go#L320-L322

We should probably change `Docker Swarm` to `Docker Classic Swarm` to prevent confusion 😓 

```yaml
    description: |
      ContainerNode stores information about the node that a container
      is running on.  It's only available in Docker Classic Swarm
```

Reply:
So; the trouble is a bit with this struct, is that the Docker Engine itself will never propagate it; it's only used for Classic Swarm (which augments the Docker API); https://github.com/docker/swarm/blob/38c89321393f53deeaf3f69af596121740f06545/cluster/engine.go#L460-L468


So from that perspective, I'm a bit on the fence on this one. On the other hand; it's defined in the API, so 🤷‍♂